mod-record-specifications-server/src/test/java/org/folio/api/SpecificationStorageSyncApiIT.java

```diff
@@ -81,22 +83,22 @@ void syncSpecification_produceSameEntitiesEachTime() throws Exception {
     var recreatedIndicatorCodes = executeInContext(() -> indicatorCodeRepository.findAll());
 
     assertThat(recreatedFields)
-      .hasSize(293)
+      .hasSize(442)
       .extracting(UuidPersistable::getId)
       .containsExactlyInAnyOrder(createdFieldIds);
 
     assertThat(recreatedSubfields)
-      .hasSize(2826)
+      .hasSize(4523)
       .extracting(UuidPersistable::getId)
       .containsExactlyInAnyOrder(createdSubfieldIds);
 
     assertThat(recreatedIndicator)
-      .hasSize(528)
+      .hasSize(816)
       .extracting(UuidPersistable::getId)
       .containsExactlyInAnyOrder(createdIndicatorIds);
 
     assertThat(recreatedIndicatorCodes)
-      .hasSize(1193)
+      .hasSize(1784)
```

Comment:
FYI: numbers have changed because specification sync runs also during the tenant setup for this test class, so number of syncs is incremented

Reply:
Could you do a database cleanup before this test or not make spec sync on tenant init. 
These numbers are representing actual numbers of entities based on test data, so would be nice to have them correct.